apps/condo/domains/organization/queries/OrganizationEmployee.graphql

```diff
@@ -138,3 +138,26 @@ query getEmployeeInvitesCount($userId: ID!) {
         count
     }
 }
+
+query getContactEditorOrganizationEmployees ($where: OrganizationEmployeeWhereInput) {
+    employees: allOrganizationEmployees (
+        where: $where,
+        first: 100
+    ) {
+        id
+        name
+        phone
+    }
+}
+
+query getOrganizationEmployeeWithTicketOrganization ($userId: ID!, $organizationId: ID!) {
```

Comment:
mb getOrganizationEmployeeByTicketOrganization or smth? From title I expect the answer to be organization, it's a bit confusing